spec/integration/kibana/tags.spec.yml

```diff
@@ -6,45 +6,46 @@ spec:
   type: array
   description: Tags automatically added to saved object assets.
   items:
-     type: object
-     additionalProperties: false
-     properties:
-       text:
-         description: Tag name.
-         type: string
-       asset_types:
-         description: >
-           This tag will be added to all the assets of these types included in the package.
-           If two or more pacakges define the same tag, there will be just one tag created in Kibana and
-           all the assets will be using the same tag.
-         type: array
-         items:
-           type: string
-           enum: # added the same types that are allowed under kibana folder
-             - dashboard
-             - visualization
-             - search
-             - map
-             - lens
-             - index_pattern
-             - security_rule
-             - csp_rule_template
-             - alerting_rule_template
-             - ml_module
-             - osquery_pack_asset
-             - osquery_saved_query
-       asset_ids:
-         description: >
-           Asset IDs where this tag is going to be added.
-           If two or more pacakges define the same tag, there will be just one tag created in Kibana and
-           all the assets will be using the same tag.
-         type: array
-         items:
-           type: string
-     oneOf:
-       - required:
+    type: object
```

Comment:
the files have been formatted ? 🤔  its hard to follow the changes purely related to the pr when all the file is changed

Reply:
do you have "Hide Whitespace" enabled?

<img width="487" height="369" alt="Image" src="https://github.com/user-attachments/assets/683ef806-e6eb-48cb-bcd3-11503efb4709" />

Reply:
I'll try to disable auto format for this project and rebase